garminconnect/__init__.py

```diff
@@ -1243,6 +1246,15 @@ def get_user_profile(self):
 
         return self.connectapi(url)
 
+    def get_userprofile_settings(self):
+        """Get user settings."""
+
+        url = self.garmin_connect_userprofile_settings_url
+        L.debug("Getting userprofile settings")
```

Comment:
Should this be `logger.debug`?

Reply:
I would guess so, as it is in almost every other function.